Adjust user avatar size to align with sidebar icon dimensions

banobepascal opened the pull request:
Change-Id: Ib7d5ae234efd9fc2d0438090f4a90e6b229b53e1


* Resolves: # <!-- related github issue -->
* Target version: master 

### Summary
Adjusts the size of the user avatar to ensure consistency with the sidebar icon dimensions. Previously, the avatar appeared disproportionately larger, which affected the visual alignment on the Toolbar

### TODO
<img width="324" alt="Screenshot 2024-12-09 at 03 16 14" src="https://github.com/user-attachments/assets/0571d7ee-2dcc-46a0-b811-c0ec19a87558">

- [ ] ...

### Checklist

- [x] I have run `make prettier-write` and formatted the code.
- [x] All commits have Change-Id
- [x] I have run tests with `make check`
- [x] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

banobepascal commented:
The button grid in Calc was already fixed, with the plus and minus buttons stacked vertically for clarity and the gap near the Merge Cell button was also resolved.
<img width="421" alt="Screenshot 2024-12-09 at 03 16 42" src="https://github.com/user-attachments/assets/69889b17-b04f-423b-b561-05b40a71a0ff">


box-sizing: border-box was applied to all top-right buttons.

pedropintosilva reviewed (changes requested):
![image](https://github.com/user-attachments/assets/33e326bc-680b-40b6-8000-71072895b247)

> Note: It is often useful to set box-sizing to border-box to lay out elements. This makes dealing with the sizes of elements much easier, [...]

https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing

-----

Also and since these are img elements we should 1. look at our CSS variables (sizes are set in browser/css/cool.css while colors are set in browser/css/color-palette/-dark.css)

Use grep to quickly find where a specific CSS vars or related CSS vars are being set:

```bash
grep -Ri '\--btn-.*:' ./browser/css/
./browser/css/cool.css: --btn-size: 28px;
./browser/css/cool.css: --btn-padding: 2px;
./browser/css/cool.css: --btn-img-size: calc(var(--btn-size) - var(--btn-padding) * 2);
./browser/css/cool.css: --btn-color-colorpicker-size: 4px;
./browser/css/cool.css: --btn-img-colorpicker-size: calc(var(--btn-size) - var(--btn-padding) * 2 - var(--btn-color-colorpicker-size));
./browser/css/cool.css: --btn-size-m: 22px;
./browser/css/cool.css: --btn-img-size-m: calc(var(--btn-size-m) - var(--btn-padding) * 2);
./browser/css/cool.css: --btn-size-s: 20px;
./browser/css/cool.css: --btn-img-size-s: calc(var(--btn-size-s) - var(--btn-padding) * 2);
```

banobepascal commented:
Added `--btn-img-size-s` to avatar img.

Also I think we need to separate the css styling for the sizes of the `#userListHeader .avatar-img` from `#userlist-dropdown .avatar-img` as it doesn't align well with the avatars in the dropdown. 

Below is the styling and visual representation that fixes the alignment of the avartars and the top right buttons

```
#userListHeader .avatar-img,
#userlist-dropdown .avatar-img {
	width: var(--btn-img-size-s);
	height: var(--btn-img-size-s);
}
```

<img width="272" alt="Screenshot 2024-12-09 at 14 47 15" src="https://github.com/user-attachments/assets/0bc3a89e-c9ad-43ef-9122-23d7a1fb2964">


@pedropintosilva box-sizing: border-box for the top right icons is already set under css/notebookbar.css: 216

pedropintosilva commented:
> [...]
> Also I think we need to separate the css styling for the sizes of the `#userListHeader .avatar-img` from `#userlist-dropdown .avatar-img` as it doesn't align well with the avatars in the dropdown.

Exactly, makes sense

pedropintosilva commented on browser/css/toolbar.css:
```diff
@@ -1589,8 +1589,8 @@ menu-entry-with-icon.padding-left + menu-entry-icon.width */
 #userlist-dropdown .avatar-img {
-	width: var(--btn-size);
-	height: var(--btn-size);
+	width: var(--btn-img-size-s);
+	height: var(--btn-img-size-s);
```
why don't we use the --btn-img-size at least for the `#userListHeader .avatar-img` I still see missing the box-sizing

pedropintosilva commented:
> [...]
> @pedropintosilva box-sizing: border-box for the top right icons is already set under css/notebookbar.css: 216

`.unotoolbutton.notebookbar .unobutton` doesn't affect the img elements there:

![image](https://github.com/user-attachments/assets/ac9c39f6-e02e-4958-a64d-dd1aa99cb5b8)

banobepascal commented:
added `box-sizing: border-box` to fix user avatars alignment. 
Reverted back to `--btn-img-size` 

<img width="277" alt="Screenshot 2024-12-10 at 01 04 54" src="https://github.com/user-attachments/assets/eea8ff09-a882-45f5-8ca4-ad356c174d46">

pedropintosilva commented:
I see 2 commits where one changes the thjngs that the first changed.

pedropintosilva commented:
Best to squash those into one commit

eszkadev commented:
rebased

eszkadev commented:
@pedropintosilva build passed, commits are squashed :)

pedropintosilva reviewed (approved):
Thanks!
